### DISCUSSION TITLE
Welcome to SOLDR modules Discussions!

### DISCUSSION THREAD

**Discussion by asdek** [vxcontrol/soldr-modules#1] [General]
## 👋 Welcome everyone!
  You may using Discussions as a place to connect with SOLDR modules developers team. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  Let's start!

**Comment by 0legovich** [comment]
1. Any ideas about test files? If I want to add test for cmodule `corelator` what is the better place for this files? Examples: `corelator/1.0.0/cmodule/main.test.lua`, `corelator/1.0.0/cmodule/tests/main.test.lua`, `tests/corelator/1.0.0/cmodule/main.test.lua` etc
2. Any ideas about mocks/stubs external packages ("engine", "system" and other) for testing?
